Assets/Scripts/Combat/CombatController.cs

```diff
@@ -55,17 +55,17 @@ virtual public bool IsPC()
 
     private void RegisterMoves()
```

Comment:
I reordered these slightly - at some point we may want to improve the AI to choose special abilities based on some situational reasoning, but for now they simply choose the first one in the list that they meet the requirements for.